src/lib/mongoBackend/mongoGetSubscriptions.cpp

```diff
@@ -115,7 +115,7 @@ static void setSubject(Subscription* s, const BSONObj& r)
 
   if (r.hasField(CSUB_EXPR))
   {
-    mongo::BSONObj expression = getFieldF(r, CSUB_EXPR).Obj();
+    mongo::BSONObj expression = getObjectFieldF(r, CSUB_EXPR);
     std::string    q          = getStringFieldF(expression, CSUB_EXPR_Q);
```

Comment:
My theory is that `embeddedObject()` and `Obj()` results the same, i.e. one function is a kind of alias of the other.

The fact that test harness are ok seems to confirm this theory.